types/index.d.ts

```diff
@@ -856,10 +853,7 @@ export namespace Operation {
     options: OperationOptions.LiquidityPoolWithdraw
   ): xdr.Operation<LiquidityPoolWithdraw>;
   interface InvokeHostFunction extends BaseOperation<OperationType.InvokeHostFunction> {
-    function: xdr.HostFunction;
-    parameters: xdr.ScVal[];
-    footprint: xdr.LedgerFootprint;
-    auth: xdr.ContractAuth[];
+    functions: xdr.HostFunction[];
```

Comment:
noticed this needed to be sync'd to the new gen'd xdr , [using this updated interface from soroban-client](https://github.com/stellar/js-soroban-client/pull/77/files#diff-fff0ee1c855d7fdef5ec027268d27066a50d0c0dc4e4832b51f8a2e40129e177R83)